src/win/utils.h

```diff
@@ -20,7 +21,10 @@ namespace Utils
 
     std::unordered_map<std::wstring, std::wstring> splitData(const std::wstring &data);
 
-    std::wstring formatData(const std::vector<std::pair<std::wstring, std::wstring>> &data);
+    // Formats the launch args like this: <notificationID>;<userInfo JSON string>
+    std::wstring formatLaunchArgs(const std::wstring &notificationID, const std::wstring &userInfo);
```

Comment:
This is another thing I'm not super proud of, and another good reason to move logic to JS where it's easier to serialize/deserialize and have some compiler-time help for this data.